tracing-servlet/build.gradle

```diff
@@ -17,16 +17,42 @@
 apply plugin: 'com.palantir.external-publish-jar'
 apply plugin: 'com.palantir.revapi'
 
+versionsLock {
+    disableJavaPluginDefaults()
+}
+
 dependencies {
     // tracing is used to clear tracing state, but not exposed to other consumers
     implementation project(":tracing")
 
     // servlet-api should be the only exposed dependency
-    api "jakarta.servlet:jakarta.servlet-api"
+    api "javax.servlet:javax.servlet-api", {
+        version {
+            strictly '4.0.1'
+        }
+    }
 
     implementation 'com.palantir.safe-logging:logger'
 
-    testImplementation "io.dropwizard:dropwizard-testing"
+    testImplementation "io.dropwizard:dropwizard-testing", {
+        version {
+            strictly '2.0.10'
+        }
+    }
     testImplementation "junit:junit"
     testImplementation 'org.assertj:assertj-core'
+
 }
+
+configurations.all {
```

Comment:
This is a little funky, but it's because dropwizard brings in a transitive dependency of Jersey, so we have to force it down this way (otherwise it's pulled up through GCV). Everything needs to be pulled up except for the inject package which has a different versioning scheme.

Reply:
ouch, I should have replaced dropwizard with something like this: https://github.com/palantir/atlasdb/pull/5978
Not worth doing within this PR though

Reply:
I ended up having to just delete these tests...there were too many edge cases that ended up breaking because further in the dependency graph certain dependencies, like slf4j, hibernate, etc. were being pinned higher, and that was incompatible with these tests.